lib/fuse.c

```diff
@@ -4421,6 +4433,7 @@ static const struct fuse_opt fuse_lib_opts[] = {
 	FUSE_LIB_OPT("noforget",              remember, -1),
 	FUSE_LIB_OPT("remember=%u",           remember, 0),
 	FUSE_LIB_OPT("modules=%s",	      modules, 0),
+	FUSE_LIB_OPT("max_idle_threads=%u",   max_idle_threads, 0),
```

Comment:
As above, `clone_fd` and `max_idle_threads` should be on the same footing. So either add `clone_fd` here, or remove the `max_idle_threads` setting.